src/_actions/users.actions.js

```diff
@@ -55,17 +64,30 @@ const loadAssetAssigneeSuccess = users => ({
   payload: users
 });
 
-export const addSecurityUser = securityUser => dispatch => axios.post('/security-users/', securityUser)
-  .then((response) => {
-    dispatch({
-      type: CREATE_SECURITY_USER_SUCCESS,
-      payload: response.data
-    });
-    dispatch(updateToastMessageContent('New Security User Added Successfully', 'success'));
-  }).catch((error) => {
-    dispatch({
-      type: CREATE_SECURITY_USER_FAILURE,
-      payload: error.message
+export const addSecurityUser = securityUser => dispatch =>
+  axios.post('/security-users/', securityUser)
+    .then((response) => {
+      dispatch({
+        type: CREATE_SECURITY_USER_SUCCESS,
+        payload: response.data
+      });
+      dispatch(updateToastMessageContent('New Security User Added Successfully', 'success'));
+    })
+    .catch((error) => {
+      dispatch({
+        type: CREATE_SECURITY_USER_FAILURE,
+        payload: error.message
+      });
+      dispatch(updateToastMessageContent('Could not save Security User', 'error'));
     });
-    dispatch(updateToastMessageContent('Could not save Security User', 'error'));
-  });
+
+export const setActivePage = page => dispatch => dispatch(setActivePageSuccess(page));
```

Comment:
No need for this.  Just export `setActivePageSuccess` and use it wherever `setActivePage` will be used. Only use `dispatch` in action creators when using thunk middleware to make API calls